examples/authentication/client/build.rs

```diff
@@ -17,7 +17,7 @@
 use std::path::Path;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let proto_path = Path::new("../../../examples/authentication/proto");
+    let proto_path = Path::new("../../../oak/proto");
     let file_path = proto_path.join("authentication.proto");
```

Comment:
It feels like there should be a single canonical crate where the autogenerated code should live, rather than being regenerated in two places.  However, none of the current candidates are great:
 - `oak_runtime` would imply a dep from client->`oak_runtime`, but the latter is really supposed to be the runtime internals and private.
 - `oak_abi` is where other similar things live, but this isn't part of the ABI (it's not related to the boundary between an Oak Application and the Runtime.
 - `oak_utils` is intended for build-time deps.

So maybe just add this to the list of things to think about under #1023 for now?

Reply:
Added to #1023